framework/config/src/main/java/org/apache/cloudstack/framework/config/impl/ConfigDepotImpl.java

```diff
@@ -166,7 +166,7 @@ public ConfigurationDao global() {
         return _configDao;
     }
 
-    public ScopedConfigStorage scoped(ConfigKey<?> config) {
+    public ScopedConfigStorage findStorage(ConfigKey<?> config) {
```

Comment:
I would use something like `findScopedConfigStorage`. This `findStorage` can be confused with find storage pool.

Reply:
Thanks, "findScopedConfigStorage" is more clear.